vitest.config.ts

```diff
@@ -9,6 +9,9 @@ export default defineConfig(
     },
     test: {
       environment: "jsdom",
+      env: {
+        npm_package_json: process.env.npm_package_json ?? "./package.json",
+      },
```

Comment:
Do you know why this isn't set by `npm` itself? would it make sense to make it absolute with `process.cwd() + "package.json"`?

Reply:
Since we don't download chrome anymore, we don't need that variable either. Removed.